.buildkite/prepare-bidding.yml

```diff
@@ -0,0 +1,121 @@
+agents:
+  queue: "snapshots"
+
+env:
+  gs_bucket: gs://marinade-validator-bonds-mainnet
+  claim_type: "bidding"
+
+steps:
+  - label: ":hammer_and_wrench: :rust: Build"
+    commands:
+    - '. "$HOME/.cargo/env"'
+    - 'cargo build --release --bin bid-distribution-cli'
+    artifact_paths:
+      - target/release/bid-distribution-cli
+
+  - input: "What is the epoch to generate settlements for?"
+    fields:
+      - text: "Epoch"
+        key: "epoch"
+        format: "[0-9]+"
+    if: "build.env('EPOCH') == null"
+  - command: "echo 'Epoch is: $EPOCH'"
+    if: "build.env('EPOCH') != null"
+
+  - wait: ~
+
+  - label: ":closed_lock_with_key: Concurrency gate lock"
+    command: echo "--> Start of concurrency gate"
+    concurrency_group: 'validator-bonds/prepare-bidding'
+    concurrency: 1
+
+  - wait: ~
+
+  - label: ":floppy_disk: :arrow_left: :cloud: Downloading Settlements JSON data"
+    commands:
+    - |
+      epoch=${EPOCH:-$(buildkite-agent meta-data get epoch)}
+      buildkite-agent meta-data set epoch "$$epoch"
+    - |
+      curl -s "https://scoring.marinade.finance/api/v1/scores/sam?epoch=$$epoch" -o sam-scores.json
+      gcloud storage cp "$gs_bucket/$$epoch/stakes.json" "."
+    key: 'download-json'
+    artifact_paths:
+      - "./*.json"
+
+  - wait: ~
+
+  - label: ":scales: Evaluate Marinade Bids"
+    env:
+      WHITELIST_STAKE_AUTHORITY: stWirqFCf2Uts1JBL1Jsd3r6VBWhgnpdPxCTe1MFjrq,4bZ6o3eUUNXhKuqjdCnCoPAoLgWiuLYixKaxoa8PpiKk,ex9CfkBZZd6Nv9XdnoDmmB45ymbu4arXVk7g5pWnt3N
+    commands:
+    - 'buildkite-agent artifact download --include-retried-jobs sam-scores.json .'
+    - 'buildkite-agent artifact download --include-retried-jobs stakes.json .'
+    - 'buildkite-agent artifact download --include-retried-jobs target/release/bid-distribution-cli .'
+    - 'chmod +x target/release/bid-distribution-cli'
+    - |
+      ./target/release/bid-distribution-cli \
+        --sam-meta-collection ./sam-scores.json \
+        --stake-meta-collection ./stakes.json \
+        --marinade-fee-bps 800 \
```

Comment:
Another thing for my interest.
Here it's defined 8% of the fee for Marinade. When we calculate APY from Bonds bidding is the marinade fee subtracted from the APY that is presented at the dashboard? Could the fee be changed and thus the sync of the value here and what's presented could be good to be somehow double-checked or synchronized?

Reply:
that was a placeholder, I need to double check with lj or someone who knows what pct marinade will take. I think I can make it as a dynamic argument to the pipeline

Reply:
Yeap, ok. Would you know about the other part how the APY is calculated? Is there considered the Marinade fee to reduce the APY for stakers in the dashboard?
Or is it probably rather a question on @janlegner , right?

Reply:
I changed the `--marinade-fee-bps` after a short sync with him, but regarding APY not sure, I think he would know better.

Reply:
> Yeap, ok. Would you know about the other part how the APY is calculated? Is there considered the Marinade fee to reduce the APY for stakers in the dashboard?

This is TBD, the same way MNDE directing or re-balancing costs are not shown in the dasboard